text/0002-encrypted-memos.md

```diff
@@ -0,0 +1,188 @@
+- Feature Name: Encrypted Memos
```

Comment:
I guess I'm going to stage this PR on that PR

Reply:
I can't do that unless I make the PRs from this repo instead of my fork, and I'd rather not do that, so I think we just have to ignore that file while reviewing this PR until that PR is merged

Reply:
I guess I'm just going to make this a parallel PR, even though it doesn't make sense if the other one isn't merged